src/e2e/java/teammates/e2e/cases/e2e/StudentProfilePageE2ETest.java

```diff
@@ -0,0 +1,73 @@
+package teammates.e2e.cases.e2e;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.StudentProfileAttributes;
+import teammates.common.util.Const;
+import teammates.e2e.pageobjects.StudentHomePage;
+import teammates.e2e.pageobjects.StudentProfilePage;
+import teammates.e2e.util.BackDoor;
+import teammates.e2e.util.TestProperties;
+
+/**
+ * SUT: {@link teammates.common.util.Const.WebPageURIs#STUDENT_PROFILE_PAGE}.
+ */
+public class StudentProfilePageE2ETest extends BaseE2ETestCase {
+
+    @Override
+    protected void prepareTestData() {
+        testData = loadDataBundle("/StudentProfilePageE2ETest.json");
+
+        String studentGoogleId = TestProperties.TEST_STUDENT2_ACCOUNT;
```

Comment:
I don't think any of the already-migrated E2E tests need real data, but let's keep it for another day.

Reply:
Can I clarify what does don't need real data means? I will try to keep it in mind for future tests

Reply:
Real data = actual Google accounts. Even if you examined their counterparts in V6, none of the two migrated actually need real accounts.